### ISSUE TITLE
[Designer Module 1: Flexible Layout and Design] Explain contrast ratio

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-curricula#373]
Learning outcome for use of color would be that students should be able to explain what 'contrast ratio' is. Probably also needs to be something about how they would go about assessing contrast ratio.

**Comment by daniel-montalvo** [comment]
Hey @iadawn 

I took a pass at
https://github.com/w3c/wai-curricula/pull/389/commits/c3e487c6afdcbc1ec506549b18da64d29e6f4e7e

I introduced the term "sufficient color contrast" and then I explain that this translates to WCAG "contrast ratio".

Do you think this deserves further explanations, like adding a specific definition?

**Comment by iadawn** [comment]
Hi @daniel-montalvo,

My concern is that the learning outcomes do not include anything about assessing contrast ratio. For a user to "design user interfaces with sufficient color contrast for text, images of text, and controls" they will need to know how to determine whether or not the color contrast is sufficient.

**Comment by daniel-montalvo** [comment]
Included in [Module Visual Design, Topic Color](https://wai-curricula.netlify.app/curricula/designer-modules/visual-design/#topic-color), learning outcomes for topic, second bullet:
"determine sufficient color contrast for text, images of text, and controls"
